lib/vagrant/cli.rb

```diff
@@ -30,6 +31,20 @@ def execute
         return 0
       end
 
+      if @main_args.include?("-install-autocomplete")
```

Comment:
Oops...just thought of this. We should guard against this flag being valid for anything other than something like `vagrant --install-autocomplete` Otherwise this will work if you do stuff like `vagrant up --install-autocomplete`?

Also this should start with at `--` instead of a single `-`.

Reply:
Also, based in what was mentioned in slack, it seems like this would be better implemented as its own Command class, rather than here.